whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -112,7 +137,101 @@ <h3 class="title">Products</h3>
 </div>
 {% endblock content %}
 {% block footer %}
+    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
```

Comment:
It works after removing this line, so I guess we can stick to the version we are using.